### PR TITLE
New version of the expandable ad template (v3)

### PR DESCRIPTION
**Pull request by tijanamiletic**
A new version of the expandable ad template has been created with updated options for parallax scrolling to match the new fluid 250 and the new scrollable MPU.

Closed:

screen shot 2015-05-28 at 10 11 44

Open:

screen shot 2015-05-28 at 10 12 09

The new version also fixes a bug, so that the ad expands only once upon scrolling into view the first time before the cookie is set, and the show more icons animate only when the ad is expanding.

An additional short 'bounce' animation was added for the down arrow when the ad scrolls into view the first time after the cookie has been set (the user has already previously seen the ad).

The other changes are small improvements to the sharpness and position of the 'show more' icons.

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v3.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+	detect,
+    mediator,
+    storage,
+    template,
+    svgs,
+    expandableV3Tpl
+) {
+
+    /**
+     * https://www.google.com/dfp/59666047#delivery/CreateCreativeTemplate/creativeTemplateId=10028247
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v3.js]: is the link to the template correct?

Comment (tijanamiletic) [inline static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v3.js]: Yes, this is referenced as expandableV3Tpl further down in the same file:

$expandableV3 = $.create(template(expandableV3Tpl, ...

**Comment by Calanthe** [comment]
:+1: looks good

**Comment by uplne** [comment]
You might want to merge master first :+1:

**Pull request merged**